CMakeLists.txt

```diff
@@ -3,6 +3,8 @@ cmake_minimum_required(VERSION 3.14)
 project(lei3d)
 
 set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON CACHE INTERNAL "")
```

Comment:
lmao I was trying to figure out how to do this a while back, the clangd extension on vscode was still giving me problems despite doing this though? Is it working for you?